repos/system_upgrade/el7toel8/actors/checkosrelease/libraries/library.py

```diff
@@ -13,10 +11,10 @@
 
 def skip_check():
     """ Check if an environment variable was used to skip this actor """
-    if os.getenv('LEAPP_SKIP_CHECK_OS_RELEASE'):
+    if os.getenv('LEAPP_DEVEL_SKIP_CHECK_OS_RELEASE'):
```

Comment:
@AloisMahdal heads up for the change of the variable name

Reply:
Just pointing out that we had discussions about the names recently and currently there are thoughts about: `LEAPP_DEVEL_UNSUPPORTED` prefix. Keeping that opened until we finish the discussion. I guess, that should happen this tuesday.